squire_web/src/tournament/players/selected.rs

```diff
@@ -20,15 +20,39 @@ impl SelectedPlayer {
     }
 
     pub fn view(&self, tourn: &Tournament) -> Html {
-        let txt = self
+        let returnhtml = self
             .id
             .map(|id| {
                 tourn
                     .get_player(&id.into())
-                    .map(|plyr| plyr.name.as_str())
-                    .unwrap_or("Player not found!!!")
+                    .map(|plyr| {
+                        html! {
+                            <>
+                            <h4>{ plyr.name.as_str() }</h4>
+                            <p>{ format!("Gamertag : {}", plyr.game_name.clone().unwrap_or_else(|| "None".to_string())) }</p>
```

Comment:
This is fairly small, but there are a lot of long function chains here (which is good), but the Rust format has a hard time formatting these chains inside the `html` macro. This is mostly stylistic, but I could move these longer function chains to the outside of the macro, assign it to a variable, and then use the variable in the macro. Not a super big deal, just a code cleanliness change.